src/widgets/roller/lv_roller_private.h

```diff
@@ -33,7 +33,7 @@ struct lv_roller_t {
     uint32_t sel_opt_id;          /**< Index of the current option*/
     uint32_t sel_opt_id_ori;      /**< Store the original index on focus*/
     uint32_t inf_page_cnt;        /**< Number of extra pages added to make the roller look infinite */
-    lv_roller_mode_t mode : 1;
+    lv_roller_mode_t mode;
```

Comment:
Would this work?
```suggestion
    lv_roller_mode_t mode :2;
```

Just to save some memory.